### PR TITLE
Added first quick draft of php uk conference blogpost

### PR DESCRIPTION
**Pull request by hannesvdvreken**
### Added

- PHP UK conference blogpost

### CODE REVIEW

File: _posts/2017-02-17-php-uk-conference.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+Here's our recap of this year's event:
+
+### Speakers dinner
+
+Leading up to the event, we received regular updates via email from Johanna, one of the main organisers.
+She was very kind, personal and offered Frederick to join me to the speakers dinner so he wouldn't "be alone".
+
+I was baffled by the hospitality.
+Of course Frederick accepted the invitation and I took Frederick as my +1 :wink:. Thanks, Johanna!
```
Comment (batusa) [inline _posts/2017-02-17-php-uk-conference.md]: Of course Frederick accepted the invitation and I took **him** as my +1...

Comment (batusa) [inline _posts/2017-02-17-php-uk-conference.md]: so that we don't repeat Frederick all the time

File: _posts/2017-02-17-php-uk-conference.md [hunk 2]
```diff
@@ -100,6 +101,25 @@ PHP user group organisers.
 
 https://twitter.com/phpgent/status/832615210250858496
 
+**Frederick:**
+Damn mornings, they seem shorter and for some reason I have a headache ... probably something in the water.
+
+Ok back to the talks!
```
Comment (batusa) [inline _posts/2017-02-17-php-uk-conference.md]: I'd put a comma after "Ok": _Ok, back to the talks!_

File: _posts/2017-02-17-php-uk-conference.md [hunk 3]
```diff
@@ -100,6 +101,25 @@ PHP user group organisers.
 
 https://twitter.com/phpgent/status/832615210250858496
 
+**Frederick:**
+Damn mornings, they seem shorter and for some reason I have a headache ... probably something in the water.
+
+Ok back to the talks! 
+
+First up: **I Think I Know What You’re Talking About, But I’m Not Sure** with [Jennifer Wong](https://twitter.com/mybluewristband) at the helm. A talk about how we live in confusing industry with a lot of confusing acronyms and how it's ok to not understand everything. The talk itself was very funny but I felt it kinda lacked a real point.
```
Comment (batusa) [inline _posts/2017-02-17-php-uk-conference.md]: _A talk about how we live in **a** confusing industry_

File: _posts/2017-02-17-php-uk-conference.md [hunk 4]
```diff
@@ -100,6 +101,25 @@ PHP user group organisers.
...
+
+My friend [Freek Van der Herten](https://twitter.com/freekmurze) is about to give his talk. But I'm a bad friend, so I went and saw [Luís Cobucci ](https://twitter.com/lcobucci) talk about **JWT - To authentication & beyond!** I really learned some cool things about JWT, those things are apparently a lot more versatile as I thought.
```
Comment (batusa) [inline _posts/2017-02-17-php-uk-conference.md]: perhaps better: _so I **left** and **watched** [Luís Cobucci ](https://twitter.com/lcobucci)_

Comment (batusa) [inline _posts/2017-02-17-php-uk-conference.md]: btw he's the author of a famous jwt lib https://github.com/lcobucci/jwt

File: _posts/2017-02-17-php-uk-conference.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+
+Just one talk left ...
+
+### Closing the conference
+
+The closing keynote was delivered by [Michael Cullum](https://twitter.com/michaelcullumuk), one of the PHP-FIG's secretaries.
+He talked about how composer and the PHP-FIG's PSR (PHP Standard Recommendations) helped evolve applications to be more framework-less.
+
+After that, the conference ended and we headed out to the pub (what else?). We had a lot of conversations with interesting people and
+talked about what they do and how they contribute to the community. I also met Korvin Szanto, one of future the PHP-FIG
```
Comment (JulieVdBroeck) [inline _posts/2017-02-17-php-uk-conference.md]: Korvin Szanto, one of the future PHP-FIG ...

File: _posts/2017-02-17-php-uk-conference.md [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+secretaries. We talked about the future of the PHP-FIG, some drafted and upcoming PSRs and how they help us evolve away from fixed frameworks and to more agnostic libraries.
```
Comment (JulieVdBroeck) [inline _posts/2017-02-17-php-uk-conference.md]: I'd leave out the 'and' in this sentence: evolve (away) from fixed frameworks to more agnostic libraries.

File: _posts/2017-02-17-php-uk-conference.md [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+Some people were even standing in the back of the room and in the hallway to see my talk.
+
+Then it was GO time! I'll paste the YouTube link here when the organisers finished uploading the footage ;-)
+
+After that I was emotionally drained because I left all my energy in the talk delivery.
+I should really thank the audience because they were really setting the right atmosphere.
+Speaking in front of a British crowd is something!
+
+The rest of the conference was back to relaxing and chatting with more interesting people.
+In the next tea break (Yeah, gotta drink tea in the UK.) we had a picture taking with all attending
```
Comment (JulieVdBroeck) [inline _posts/2017-02-17-php-uk-conference.md]: you mean someone took a pic of all you guys together, right? Then it's " we had a/our picture taken.."

Comment (hannesvdvreken) [inline _posts/2017-02-17-php-uk-conference.md]: nice catch, stupid mistake by me

**Pull request merged**